src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.jupiter.api.Test;
+
+import tasks.TaskList;
+import tasks.ToDos;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+
+    @Test
+    void getNumOfTasks() {
+        TaskList task = new TaskList();
+        task.addTask(new ToDos("do 2103"));
+        task.addTask(new ToDos("do 2101"));
+        assertEquals(1, task.getNumOfTasks());
```

Comment:
I am confused by this part, shouldn't the expected output of `task.getNumOfTasks()` be 2?